doc/FLINT models.md

```diff
@@ -0,0 +1,404 @@
+# FLINT models
+
+This documentation is a developer reference to create FLINT data models in JSON-LD format. Please see the [flinteditor-mps](https://github.com/discipl/flinteditor-mps) for more end-user oriented way to build FLINT models.
+
+## Model definition
+
+A FLINT model is a JSON-LD representation of a process. It describes actions that actors can (or must) take, conditions that must be fulfilled in order to take an action and contains official jurisdictional references to law and regulation texts. Each part of the model is saved as a verifiable claim and actions that are taken on the model will be expressed in new claims. The discipl-law-reg API can be used to play a certain role in the defined process and can perform actions as a certain actor.
+
+A basic FLINT model is a JSON object that consists out of four properties:
+
+1. `model` - A unique description that describes the processes that is modeled by the FLINT model
+2. `acts` - An array of **actions** that may be taken during evaluation of the process
+3. `facts` - An array of **facts** that can be referenced within the model
+4. `duty` - An array of **duties** that can describe agreements between actors and enforce certain actions when this duty is broken
+
+### Model
+
+The `model` property contains a description that describes the processes that is modeled by the FLINT model.
+
+
+```json
+{
+    "model": "Fictieve kinderbijslag"
+}
+```
+### Acts (actions)
+
+The `acts` property contains an array of actions that may be **taken** during the process. An act is defined by an object that describes among others: the actor that will take the action; the recipient that receives the result; the conditions that must be met in order to take the act.
+
+> On the implementation level the `recipient` expresses a need (with help of the [discipl-abundance-service](https://github.com/discipl/discipl-abundance-service/)) and the `actor` attends to this need in order to fulfill it. So the actors in the model are referring to entities in the abundance service.
+
+An act consists out of the following properties:
+
+- `act` - A unique sentence that describes the act and is used to reference the action within the model. This sentence is usually enclosed between `<<` and `>>` to indicate that the sentence is referring to an act.
+- `actions` - One or few words denoting what action is done on the object.
+- `actor` - A reference to a fact that defines the actor that can take this action.
+- `recipient` - A reference to a fact that defines the recipient of the result of this action.
+- `object` - The object on which the action is executed by the actor, and that is needed by the recipient of this action. The object can be a entity or agent or a reference to such via a fact.  The object can also be a reference to any other fact or even an action and duty referring to their corresponding state.
```

Comment:
What do you mean by the last sentence here?

Reply:
I actually see this was in the `vision` document. I'm not sure it's true. In any case, we don't support it at the moment, so let's leave it out

Reply:
I removed the last sentence :+1: